tests/html/base/load-base.html

```diff
@@ -1,65 +0,0 @@
-<!doctype html>
```

Comment:
this file was testing `document.baseURI` which is not polyfilled by `html-imports`